ScienceLogic/revis-network
# Add customControls prop and example usage

This adds the `customControls` prop to allow developers to render custom control buttons, while hooking into the same functionality used by the default control buttons.

![revis-custom-controls](https://user-images.githubusercontent.com/2181467/70180376-7feee900-16a5-11ea-9a27-946d65436491.gif)

## Testing
1. Run the following script:
```
git pull;
git checkout feat/SLUI-5967-maps-theming;
yarn start
```
3. Open http://localhost:3000/
4. Verify the default buttons behave as intended.
5. Open `src/examples/index.js` and change the following import:
```
import Example from './Force2Example';
```
to:
```
import Example from './CustomControlsExample';
```
6. Refresh http://localhost:3000/
7. Verify the custom buttons behave as intended.
